cloudformation/cf10aws.yml

```
@@ -0,0 +1,353 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Resources: 
+
+# Vpc,create
+  Vpc10:
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock: 10.10.0.0/20
+      EnableDnsSupport: true
+      Tags:
+        - Key: Name
+          Value: Vpc10cf
```

パスカルケースに統一するなら、以下のようになるかなと！
```suggestion
          Value: Vpc10Cf
```
他にもありそうなのでもう一度確認してみてください！

---

ありがとうございます！
確認して提出します！